core/templates/dev/head/pages/library/library.html

```diff
@@ -1,13 +1,5 @@
 {% extends 'pages/base.html' %}
 
-{% block maintitle %}
-  {% if page_mode in ['group', 'search'] %}
```

Comment:
Can we remove the page_mode from the handler if it's no more required?

Reply:
Hi I can't find page_mode in the library handler. However, search_mode is used to set the description of the page.

Reply:
Thanks for checking, so you mean this wasn't working as expected earlier?

Reply:
It was added [here](https://github.com/oppia/oppia/blob/302fa1d9853a54986de3a75e7d56cf08c55f02bc/core/controllers/library.py#L92), however, i that is from 3 years ago. It isn't there at the moment. But it was working as expected earlier.

Reply:
I think this issue was introduced in #6415, thanks for checking!

Reply:
Okay, cool. Thanks